tgui-next/packages/tgui/interfaces/Autolathe.js

```diff
@@ -0,0 +1,416 @@
+                                )}
+                              </Box>
+        
+                            )}
+        
+                          </Grid.Column>
+                        </Grid>
+                      </div>
+                    ))}
+      
+                  </Flex.Item>
+                </div>
+              </Section>
+            ) : (
+              <Section fluid title="Known Designs" width={100}>
+                <div>
+                  <Flex.Item>
+                    {data.designs.filter(design => {
+                      return (design.category.includes(data.selected_category));
+                    }).map(design => (
+                      <div key={data.designs}>
+                        <Grid>
+                          <Grid.Column size={2.5}>
+                            <Button
+                              inline
+                              key={design.name}
+                              content={design.name}
+                              disabled={design.disabled}
+                              title={design.name}
+                              mr={1}
+                              icon="print"
+                              onClick={() => act(ref, 'make', {
+                                item_id: design.id,
+                                multiplier: 1,
+                              })} />
+              
+                            {design.max_multiplier.map(max => (
+                              <Button
+                                inline
+                                key={max}
+                                disabled={design.disabled}
+                                content={max + "x"}
+                                onClick={() => act(ref, 'make', {
+                                  item_id: design.id,
+                                  multiplier: max,
+                                })}
+                              />
+                            ))}
+                          </Grid.Column>
+                          <Grid.Column size={1}>
+                            {design.materials_metal === 0 ? (
+                              ""
+                            ):(
+                              <Box ml={0} mr={0} inline
+                                color={(
+                                  data.metal_amount >= design.materials_metal ? 'white' : 'bad'
+                                )}>
+                                {data.metal_amount >= design.materials_metal ? (
+                                  <div>Metal: {design.materials_metal}</div>
+                                ) : (
+                                  <b>Metal: {design.materials_metal}</b>
+                                )}
+                              </Box>
+
+                            )}
+            
+                          </Grid.Column>
+                          <Grid.Column size={1}>
+                            {design.materials_glass === 0 ? (
+                              ""
```

Comment:
Can't you just make it display only if design.materials_glass > 0

Reply:
This breaks it.

Reply:
How does it break?

Reply:
that's because gamer's a dumb-dumb and it should be <= 0.... unless you flip the two sides of the :

Reply:
Meh don’t do programming at midnight, thanks monster

Reply:
But shouldn’t it display if the design costs more than 0 glass, I’m confused
We’re trying to remove the empty element

Reply:
Its fine I fixed it by flipping it.